luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy

```diff
@@ -0,0 +1,28 @@
+package com.yahoo.bard.webservice.config.luthier
+
+
+import com.yahoo.bard.webservice.config.luthier.factories.KeyValueStoreDimensionFactory
+import com.yahoo.bard.webservice.data.config.ResourceDictionaries
+import com.yahoo.bard.webservice.data.dimension.Dimension
+import spock.lang.Specification
+
+class LuthierIndustrialParkSpec extends Specification {
+    LuthierIndustrialPark industrialPark
+    void setup() {
+        Map<String, Factory<Dimension>> dimensionFactoriesMap = new HashMap<>()
+        dimensionFactoriesMap.put("KeyValueStoreDimension", new KeyValueStoreDimensionFactory())
+
+        ResourceDictionaries resourceDictionaries = new ResourceDictionaries()
+        industrialPark = new LuthierIndustrialPark.Builder(resourceDictionaries)
+                .withDimensionFactories(dimensionFactoriesMap)
+                .build()
+
+    }
+
+    def "An industrialPark instance is loaded up without error."() {
+        when:
+            industrialPark.load()
+        then:
+            true
+    }
```

Comment:
Additional tests to write:
1- test that retrieving an already created dimension returns the same instance (e.g. isA()  in groovy )
2- test that building a dimension that doesn't have config results in the correct error.
3- test that building a dimension whose config refers to an invalid dependency has the correct error.

Reply:
Some of these are addressed, again, in the new branch and eventually in the next PR, which will wrap up the dimension's implementation. I think it is not advisable to snowball this PR to something uncontrollable and we should close this without more features.

Reply:
Great suggestions about the tests, and I will definitely include them soon.